index.js

```diff
@@ -10,122 +10,122 @@ var url = require('url')
 module.exports = Github
 
 function Github (opts) {
-  if (!(this instanceof Github)) return new Github(opts)
+  var ctx = {}
   opts = opts || {}
 
   assert.equal(typeof opts, 'object', 'township-github: opts should be type Object')
 
-  this.secret = opts.GITHUB_SECRET || opts.secret
-  this.name = opts.GITHUB_NAME || opts.name
-  this.id = opts.GITHUB_ID || opts.id
+  ctx.secret = opts.GITHUB_SECRET || opts.secret
+  ctx.name = opts.GITHUB_NAME || opts.name
+  ctx.id = opts.GITHUB_ID || opts.id
```

Comment:
hm could we rename all of these to just `var foo =` style? - no reason to hold them on an object anymore